rfd/0098-securing-docker-images.md

```diff
@@ -0,0 +1,266 @@
+---
+authors: Trent Clarke (trent@goteleport.com)
+state: draft
+---
+
+# RFD 0098 - Delivering secure Teleport OCI Images
+
+# Required Approvers
+* Engineering: @r0mant
+* Security: @reedloden
+* Product: (@xinding33 || @klizhentas)
+
+## What
+
+This RFD discusses structures and processes to increase the security of the OCI 
+images we use to deliver Teleport to our customers.
+
+## Why
+
+One of our shipping atrifacts is are a collection of OCI images. As the
+provider of those OCI images we are (at least partialy) responsible for
+everything in it, not just Teleport. 
+
+We should not ship vulnerabilities to our clients, even if those 
+vulnerabilities are not directly in our software.
+
+## Details
+
+### Goals
+
+For the sake of this RFD, I will define _delivering a secure image_ as
+
+> Reliably producing an OCI image that is _a priori_ is unlikely to contain 
+> vulnerabilities in and of itself, in that it:
+>
+>  1. has the smallest footprint reasonably possible,
+>  2. contains software of known provenance,
+>  3. has no warnings or vulnerabilities flagged when run through a reputable 
+>     scanner at the time of creation,
+>  4. flags vulnerabilities even after creation, either in Teleport itself or 
+>     a dependency, and
+>  5. is updated to resolve any discovered vulnerabilities in a timely fashion.
+
+For the purposes of this discussion, a "timely fashion" for updates is as the
+Teleport Vulberability policy:
+
+   | Level    | Response time |
+   |----------|---------------|
+   | Critical | 7 days        |
+   | High     | 30 days       |
+   | Moderate | 90 days       |
+   | Low:     | ~180 days     |
```

Comment:
```suggestion
   | Low      | ~180 days     |
```

Reply:
To be fair, no set time on low severity issues, but would be nice if fixed within 180 days.